src/main/resources/i18n/aerospace/common.properties

```diff
@@ -170,7 +170,8 @@ TESTING=Testing...
 VALID_VIRTUALIZE_SERVER_URL=Valid URL.
 INVALID_VIRTUALIZE_SERVER_URL=Invalid URL.
 INVALID_PARASOFT_VIRTUALIZE_SERVER_URL_1=Can't establish connection with Virtualize server.
-INVALID_KAFKA_SERVER_URL=Can't establish connection with kafka server.
+INVALID_KAFKA_SERVER_URL=Can't establish connection with Kafka broker.
+CONNECT_KAFKA_BROKER_SUCCESS=Connect kafka broker successfully.
```

Comment:
Please change "kafka" to "Kafka" and change this message to: "Connection to Kafka broker established successfully."